lib/capistrano/tasks/npm.rake

```diff
@@ -10,12 +10,13 @@ namespace :npm do
           set :npm_flags, '--production --silent --no-spin'
           set :npm_roles, :all
           set :npm_env_variables, {}
+          set :npm_method, 'install'
     DESC
   task :install do
     on roles fetch(:npm_roles) do
       within fetch(:npm_target_path, release_path) do
         with fetch(:npm_env_variables, {}) do
-          execute :npm, 'install', fetch(:npm_flags)
+          execute :npm, fetch(:npm_method, 'install'), fetch(:npm_flags)
```

Comment:
There's a place at the bottom of this file to define defaults which is more consistent I guess.

Reply:
Yes I spotted that. Have made a PR #25 in case the original submitter doesn't respond.

Reply:
Thanks for the pointer! Have updated the PR.